gitlabform/__init__.py

```diff
@@ -55,6 +55,7 @@ def __init__(
         config_string=None,
         noop=False,
         output_file=None,
+        recurse_subgroups=True,
```

Comment:
should default to False?

Reply:
depends on whether you want to make this the default (for cli) in the future. The tests currently pass with either True or False. If you want to go with True in the future (IMO you should) then i would leave it like that.